src/arrayMethodSort.js

```diff
@@ -3,10 +3,40 @@
 /**
  * Implement method Sort
  */
+function compare(A, B) {
+  const a = '' + A;
+  const b = '' + B;
```

Comment:
I noted that you've changed the function name as well as the arguments. I would believe that keeping the function name would be better as well as the argument names.